modules/commons/src/main/resources/schemas/neurosciencegraph/commons/protocol/v0.1.5.json

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/commons/protocol/v0.1.4/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0",
+    "{{base}}/schemas/nexus/schemaorgsh/identifier/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/organization/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/person/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:ProtocolShape",
+      "@type": "sh:NodeShape",
+      "label": "Protocol shape definition.",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/entity/v0.1.0/shapes/EntityShape"
+        },
+        {
+          "path": "schema:identifier",
+          "name": "Identifier",
+          "description": "The identifier of protocol such as a DOI, ISBN,...",
+          "node": "{{base}}/schemas/nexus/schemaorgsh/identifier/v0.1.0/shapes/IdentifierShape"
```

Comment:
Refer to the latest commons/identifier schema

Reply:
I think this might be the latest already? https://github.com/INCF/neuroshapes/blob/master/modules/nexus-schemas/src/main/resources/schemas/nexus/schemaorgsh/identifier/v0.1.0.json